### ISSUE TITLE
Feature request: middle mouse button scrolling

### ISSUE THREAD

**Issue by Kebap** [edbee/edbee-lib#129]
Can we have some help to quickly scroll editor with just one mouse drag?

- Say a document is too large for one editor view
- Press and hold middle mouse button
- No scrolling available today
- After feature has been added, dragging up/down will scroll editor pane up/downwards. Dragging left/right will do same sideways.

Try it in your friendly browser window for comparison. E.g. with a long page document, or in an editor area that holds too much content.

**Comment by gamecreature** [comment]
@Kebap I don't see this behaviour anywhere in any app on my Mac? 
For example VSCode, middle mouse dragging results in column-selection.  
Is this Windows specific?

**Comment by Kebap** [comment]
Yeah, after some web searches, this indeed seems Windows specific, where it's available on operating-system-level, even though many applications (like browsers, text editors, etc.) seem to implment it on an per-app-level as well to function cross-platform.

Some related pages I found:
- https://www.how2shout.com/how-to/scroll-up-down-left-and-right-automatically-pc-or-laptop.html - more thorough explanation of the feature including a gif of it in action
- https://unix.stackexchange.com/questions/58900/how-to-scroll-the-screen-using-the-middle-click/421672#421672 - looking for a solution at OS-level for Linux 
- https://superuser.com/questions/303424/can-i-enable-scrolling-with-middle-button-drag-in-os-x - looking for a solution at OS-level for Mac OS X
- https://www.reddit.com/r/mac/comments/8rs8ti/get_windows_middle_click_function_in_macos/ - looking to remove middle-mouse-pasting while keeping middle-mouse-scroll for Linux

**Comment by gamecreature** [comment]
I don't think this is very important to implement for now in edbee itself. 
- behaviour is non-standard (other compontents/widgets don't behave like this)
- mouse wheels are very common and are supported 
- Maybe other middle-mouse button functionality should be implemented in edbee (don't really know what 'the' standard is for middle mouse button)
